pennylane/devices/tests/test_gates.py

```diff
@@ -79,11 +79,20 @@
     "DoubleExcitationPlus": qml.DoubleExcitationPlus(0, wires=[0, 1, 2, 3]),
     "DoubleExcitationMinus": qml.DoubleExcitationMinus(0, wires=[0, 1, 2, 3]),
     "QubitCarry": qml.QubitCarry(wires=[0, 1, 2, 3]),
-    "QubitSum:": qml.QubitSum(wires=[0, 1, 2]),
+    "QubitSum": qml.QubitSum(wires=[0, 1, 2]),
+    "PauliRot": qml.PauliRot(0, "XXYY", wires=[0, 1, 2, 3]),
+    "U1": qml.U1(0, wires=0),
+    "U2": qml.U2(0, 0, wires=0),
+    "U3": qml.U3(0, 0, 0, wires=0),
```

Comment:
Note that these operations are not actually supported by `default.qubit`, but they could be in future.

Reply:
Hmm, and they get decomposed too, right?

Reply:
For `default.qubit` they will be decomposed. For the dev test matrix, it'll only be used if the device supports it since we have the `--skip-ops` flag everywhere (I think?).

Reply:
As an aside, `default.qubit` probably should support the `U1` etc. directly right?